blazor/datagrid/virtualization.md

```diff
@@ -167,6 +167,77 @@ To setup the column virtualization, set the [EnableVirtualization](https://help.
 
 > The collapsed/expanded state will persist only for local dataSource while scrolling.
 
+### Autofit columns with column virtualization
```

Comment:
There is no need to go for separate topic in limitation document remove the autofil that is enough